src/main/java/wolox/training/controllers/BookController.java

```diff
@@ -0,0 +1,18 @@
+package wolox.training.controllers;
+
+import org.springframework.stereotype.Controller;
+import org.springframework.ui.Model;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.RequestParam;
+
+@Controller
+public class BookController {
```

Comment:
Just a comment, since this controller has the only purpose of greeting, the name should reflect this purpose, so it could be something like GreetingController. You can leave this as is because in the future you'll be adding Book CRUD operations, this is simply a reminder 😊